test/Monotonicity.lean

```diff
@@ -3,12 +3,12 @@ Copyright (c) 2019 Simon Hudon. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Simon Hudon
 -/
-import Mathlib.Tactic.Monotonicity
-import Mathlib.Tactic.NormNum
 import Mathlib.Algebra.Order.Ring.Defs
--- import measure_theory.measure.lebesgue
--- import measure_theory.function.locally_integrable
 import Mathlib.Data.List.Defs
+import Mathlib.Tactic.Monotonicity
+import Mathlib.Tactic.NormNum
+-- import Mathlib.MeasureTheory.Function.LocallyIntegrable
+-- import Mathlib.MeasureTheory.Measure.Lebesgue.Integral
```

Comment:
Just remove?

Reply:
I moved them next to the commented example where they would be useful.  I think that rather than removing them, this might make it simpler, if/when the example works again.